include/graphblas/nonblocking/blas3.hpp

```diff
@@ -571,6 +571,110 @@ namespace grb {
 		);
 	}
 
+	template<
+		Descriptor descr = descriptors::no_operation,
+		typename InputType, typename OutputType,
+		typename RIT_L, typename CIT_L, typename NIT_L,
+		typename RIT_A, typename CIT_A, typename NIT_A
+	>
+	RC tril(
+		Matrix< OutputType, nonblocking, RIT_L, CIT_L, NIT_L > &L,
+		const Matrix< InputType, nonblocking, RIT_A, CIT_A, NIT_A > &A,
+		const long int k,
+		const Phase &phase = Phase::EXECUTE,
+		const typename std::enable_if<
+			!grb::is_object< OutputType >::value &&
+			!grb::is_object< InputType >::value &&
+			std::is_convertible< InputType, OutputType >::value
+		>::type * const = nullptr )
+	{
+#ifdef _DEBUG
+		std::cout << "In grb::tril( nonblocking )\n";
+#endif
+		// nonblocking execution is not supported
```

Comment:
please print a warning whenever a blocking placeholder implementation is used (as done e.g. in the present "nonblocking" mxm)